movie/src/components/Navbar/index.jsx

```diff
@@ -17,6 +18,17 @@ const Navbar = () => {
   const isHumbergerIcon = useIsOnMobile();
   const onMobile = useIsOnMobile();
   const [isMenuOpen, setIsMenuOpen] = useState(false);
+  const rightMenu = [
```

Comment:
what is rightMenu? 😅

Reply:
rightMenu means the menu at the right side.

Reply:
Still...what is `rightSideMenu` ? 😅 

And what if we add 3 more menus to the right side,  what would that lead to 😅

Reply:
*** rightSideMenu : I mean menu at the right side of menu bar. 
*** I added only HumbergerIcons to the rightSideMenu. I did like this because HumbergerIcons (HumbergerNavbar) will appear on the right of mobile screen.